### PR TITLE
Dependencies for Docusaurus

### PR DESCRIPTION
**Pull request by nysdas**
Added(1):
	setup.sh
	purgeCommands.sh
Modified(2):
	ReadME.md

Details:
1.
	- 'setup.sh' script will install all the dependencies for CSS Wiki App.
	- 'purgeCommands.sh` will purge all the dependencies we installed
	for the CSS Wiki

2.
	- Updated details how to run the script.

### Did you read and follow article requirements
Yes
### What are you trying to accomplish?
Updated the information: How to setup the initial environment.

### How are you accomplishing it?
Updating the README files

### Is there anything reviewers should know?
I have detailed information in the commit itself.

- [x] Is it safe to rollback this change if anything goes wrong?
- [x] yes

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (prairir) [inline setup.sh]: I dont think we should have a setup script. I think we should instead link to the official install resources.

Ubuntu is known for having out of date repos and I do not want to promote having out of date software.

File: purgeCommands.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+# This script will purge all the dependencies which were
+# required to run the CSS Wiki Application
+
+
+# Purge npm packages
+sudo apt-get purge npm -y
+
+# Purge yarn packages
+sudo apt-get purge cmdtest yarn -y
```
Comment (prairir) [inline purgeCommands.sh]: Same with `./setup.sh`

Also `cmdtest` and `yarn` are different in the apt repos. So this wouldnt purge yarn but get rid of an unrelated package

File: README.md [hunk 3]
```diff
@@ -2,20 +2,50 @@
...
 
-```console
-yarn install
+* Go into Wiki directory after cloning the Repo `cd Wiki/`
+* Locate `setup.sh` script in repo
+* Run `sudo ./setup.sh` or you can install manually one after another.
+
+Make sure to run script with `sudo` privileges.
+
+```bash
```
Comment (prairir) [inline README.md:22]: I do not think we need to put the entire script in the readme

File: README.md [hunk 4]
```diff
@@ -31,3 +61,27 @@ GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
...
+
+
+## Purge Installed Packages
+
+This script will purge/delete or the packages we installed for CSS Wiki.
+
+* Locate `purgeCommands.sh` in the repo.
+	* Run with sudo privileges `sudo ./purgeCommands.sh`
+
+```bash
```
Comment (prairir) [inline README.md]: same as above

File: README.md [hunk 5]
```diff
@@ -2,20 +2,50 @@
 
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
-## Installation
+## Installation Steps for Ubuntu
```
Comment (prairir) [inline README.md]: I think its a good idea to list the dependencies above and link to the official installation guides.

I also think having a section entitled `Dependencies` and keep `Installation` the same.

**Review by prairir — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (nysdas) [reply README.md:22]: I will remove the script Fromm Readme. just making sure You want to keep the setup.sh in the repo?

Comment (nysdas) [reply README.md:22]: I didn't read your comment on the top that says you don't want script. Please ignore ^

File: setup.sh (hunk 1 above)
Comment (nysdas) [reply setup.sh]: Setup.sh is removed.

File: purgeCommands.sh (hunk 2 above)
Comment (nysdas) [reply purgeCommands.sh]: removed

File: README.md (hunk 5 above)
Comment (nysdas) [reply README.md]: Added dependencies in latest commit.

**Review by kilbouri — APPROVED** [review]
Looks fine to me.

**Comment by prairir** [comment]
@nysdas looks good but please resolve the merge conflicts before merging

**Review by prairir — APPROVED** [review]

**Pull request closed without merging**